metadata.rb

```diff
@@ -1,6 +1,9 @@
 name             'bsdcpio'
-maintainer       'Joseph Anthony Pasquale Holsten'
-maintainer_email 'joseph@josephholsten.com'
+source_url       'https://github.com/sous-chefs/bsdcpio'
+issues_url       'https://github.com/sous-chefs/bsdcpio/issues'
+maintainer       'Sous Chefs'
+maintainer_email 'help@sous-chefs.org'
+chef_version     '>= 11'
```

Comment:
If you didn't put the guard around source_url you're actually requiring Chef 12. At this point you should just require Chef 12.